lib/util/parsers.js

```diff
@@ -1,31 +1,9 @@
 import yamlit from 'js-yaml'
 import markdownit from 'markdown-it'
-import markdownAnchors from 'markdown-it-anchor'
 
```

Comment:
I like the idea to pass the configuration from the outside, but I'm currently not aware of all possible side effects.